packages/terra-form-select/src/combobox/Frame.jsx

```diff
@@ -520,18 +522,21 @@ class Frame extends React.Component {
    * Falls back to the string 'Search' if no label is provided
    */
   ariaLabel() {
-    const { ariaLabel, disabled, intl } = this.props;
+    const {
+      ariaLabel,
+      disabled,
+      intl,
+    } = this.props;
 
-    const defaultAriaLabel = intl.formatMessage({ id: 'Terra.form.select.ariaLabel' });
     const dimmed = intl.formatMessage({ id: 'Terra.form.select.dimmed' });
 
     // VO on iOS doesn't do a good job of announcing disabled stated. Here we append the phrase that
     // VO associates with disabled form controls.
     if ('ontouchstart' in window && disabled) {
-      return ariaLabel === undefined ? `${defaultAriaLabel} ${dimmed}` : `${ariaLabel} ${dimmed}`;
+      return `${ariaLabel} ${dimmed}`;
```

Comment:
what happens when ariaLabel is undefined ..?

Reply:
Updated - https://github.com/cerner/terra-core/pull/3976/commits/4a68901c81bda93178cea45b9902d18c7346a166